backend/src/main/java/de/neuefische/app/Main.java

```diff
@@ -0,0 +1,12 @@
+package de.neuefische.app;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+public class Main {
```

Comment:
Why was the class commited?

Reply:
Yes I should just delete it, I used it before to check somethin